tidb-server/main.go

```diff
@@ -479,13 +480,23 @@ func setupMetrics() {
 		if callBackCount >= 5 {
 			callBackCount = 0
 			metrics.KeepAliveCounter.Inc()
+			updateCPUUsageMetrics()
 		}
 	}
 	go systimemon.StartMonitor(time.Now, systimeErrHandler, sucessCallBack)
 
 	pushMetric(cfg.Status.MetricsAddr, time.Duration(cfg.Status.MetricsInterval)*time.Second)
 }
 
+func updateCPUUsageMetrics() {
```

Comment:
We should setup a background goroutine to update the CPU usage metrics periodically?

Reply:
Seems `go systimemon.StartMonitor(time.Now, systimeErrHandler, sucessCallBack)` achieves this?

Reply:
Sorry, I missed this function `go systimemon.StartMonitor(time.Now, systimeErrHandler, sucessCallBack)`. I think it's no need to create another goroutine. 😄